src/MailProcessor.php

```diff
@@ -0,0 +1,198 @@
+<?php
+
+namespace Civi\Emailfiling;
```

Comment:
This is a service class and as such we should move it to a proper folder structure.
i.e CRM/Emailfiling/Service/Emailfiling.php. Also we don't need to namespace the service once we move it there.